server/WebAPI/Services/CelebrityService.cs

```diff
@@ -0,0 +1,106 @@
+﻿using Entities.DataAccess;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using WebAPI.DTO;
+
+namespace WebAPI.Services
+{
+    public interface ICelebrityService
+    {
+        List<CelebrityDTO> GetCelebrities();
+        CelebrityDTO GetCelebrity(int CelebrityId);
+    }
```

Comment:
separate folder and file for interfaces or in rot and separate folder for implementations